pkg/build/docker_node.go

```diff
@@ -0,0 +1,133 @@
+package build
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"reflect"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/testground/testground/pkg/api"
+	"github.com/testground/testground/pkg/docker"
+	"github.com/testground/testground/pkg/rpc"
+)
+
+const (
+	DefaultNodeBuildBaseImage = "node:14.12.0-buster"
+)
+
+var (
+	_ api.Builder = &DockerNodeBuilder{}
+)
+
+type DockerNodeBuilder struct{}
+
+func (d DockerNodeBuilder) ID() string {
+	return "docker:node"
+}
+
+func (d DockerNodeBuilder) Build(ctx context.Context, in *api.BuildInput, ow *rpc.OutputWriter) (*api.BuildOutput, error) {
+	cfg, ok := in.BuildConfig.(*DockerNodeBuilderConfig)
+	if !ok {
+		return nil, fmt.Errorf("expected configuration type DockerNodeBuilderConfig, was: %T", in.BuildConfig)
+	}
+
+	cliopts := []client.Opt{client.FromEnv, client.WithAPIVersionNegotiation()}
+
+	var (
+		basesrc  = in.UnpackedSources.BaseDir
+		cli, err = client.NewClientWithOpts(cliopts...)
+	)
+
+	ctx, cancel := context.WithTimeout(ctx, 30*time.Minute)
```

Comment:
I see this is copied from `docker:go` builder, but we don't need it anymore, as we are always passing through the supervisor, which applies the `task timeout` (by default 10min, but configurable). So I suggest we delete this (maybe also directly delete in `docker:go` builder as well.